environment.yml

```diff
@@ -19,6 +20,8 @@ dependencies:
   - cudnn
   - pytorch::cuda100
   - matplotlib
+  - git # to download COCO dataset
```

Comment:
The COCO dataset download script uses "curl" and "unzip". Are those included in this git package or should they be added as well?

Reply:
good spot! Thanks, updated